examples/hotrod/services/driver/server.go

```diff
@@ -32,7 +32,7 @@ import (
 // Server implements jaeger-demo-frontend service
 type Server struct {
 	hostPort string
-	tracer   opentracing.Tracer
+	tracer   *trace.TracerProvider
```

Comment:
Is `trace.TracerProvider` a struct or an interface? You don't need a pointer if it's an interface.

Reply:
Sorry by mistake, I left it when I was testing around this.